app/models/druid_version_zip.rb

```diff
@@ -32,10 +32,11 @@ def base_key
   # the archival directory structure, just the object, e.g. starting at 'ab123cd4567/...' directory,
   # not 'ab/123/cd/4567/ab123cd4567/...'
   def create_zip!
+    check_moab_version_readability!
     ensure_zip_directory!
     combined, status = Open3.capture2e(zip_command, chdir: work_dir.to_s)
     raise "zipmaker failure #{combined}" unless status.success?
-    raise "zip size (#{total_part_size}) is smaller than the moab size (#{moab_size})! zipmaker failure #{combined}" unless zip_size_ok?
+    raise "zip size (#{total_part_size}) is smaller than the moab size (#{moab_version_size})! zipmaker failure #{combined}" unless zip_size_ok?
```

Comment:
love the method name change - how about adding version to the text here as well?

Reply:
Also maybe say that it the error message? "moab version size" instead of "moab size"?

Reply:
good catch, fixed!

Reply:
ah, yeah, @julianmorley that was what i interpreted @ndushay's suggestion to mean (sorry, GH didn't auto-refresh the page, and i didn't see this till i'd already pushed a change -- let me know if i missed something from either of your suggestions)